test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -610,6 +610,37 @@ describe('Dropdown', () => {
         .find('.selected')
         .should.contain.text('a2')
     })
+    it('filter after diacritics', () => {
+      const opts = [
+        { text: 'FLOREŞTI', value: '1' },
+        { text: 'ŞANŢU FLOREŞTI', value: '2' },
+        { text: 'FLOREŞTI Alba', value: '3' },
+      ]
+
+      // search for 'floresti'
+      wrapperMount(<Dropdown options={opts} search selection />)
+        .simulate('click')
+        .find('input.search')
+        .simulate('change', { target: { value: 'floresti' } })
+
+      wrapper
+        .find('.selected')
+        .should.contain.text('FLOREŞTI')
```

Comment:
This looks great.  We can end the test here as one assertion is enough to prove the feature.  

Skipping the additional arrow down behaviors will make this test stronger.  This way, the `filter after diacritics` test doesn't fail if the arrow down behavior changes, but only if the search filter behavior changes. 👍